R/utils.R

```diff
@@ -607,6 +607,18 @@ generate_js_dist_html <- function(href,
   }
 }
 
+generate_js_inline <- function(js_map) {
+  inlineJSFormatted <- paste0("<script>window.dash_clientside = window.dash_clientside || {};
+
+                              (function() {
+                              // some imaginary ES5/mergeDeep function
+                              
+                              mergeDeep(window.dash_clientside, {\n",
+                              paste0(names(js_map), ": {\n", js_map, collapse="},\n"),
+                              "});
```

Comment:
Would this be easier if, like the [Python implementation](https://github.com/plotly/dash/pull/967/files#diff-228c9aefac729977642672aa1416bacaR1242-R1248), rather than saving a map you just saved individual script tags to add?
In principle it's actually nicer this way, with all of them in one script tag, but you will have to sort it out so the result is plain old js just like on the Python side.